scripts/build-debug-linux.sh

```
@@ -0,0 +1,55 @@
+
+echo ""
+echo "Build complete. Debug binary at:"
+echo "  ${BUILD_DIR}/src/node-exe/kth"
+echo ""
+echo "To debug: gdb ${BUILD_DIR}/src/node-exe/kth"
+echo "  Or with args: gdb --args ${BUILD_DIR}/src/node-exe/kth -c config.cfg"
+
+# Run tests if specified
+if [ "$TEST" != "0" ]; then
+    echo "Running tests..."
+    cd "${BUILD_DIR}"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)